src/main/java/org/eclipse/cargotracker/interfaces/handling/rest/HandlingReportService.java

```diff
@@ -36,31 +35,24 @@ public HandlingReportService() {}
 
   @POST
   @Path("/reports")
-  @Consumes({"application/json", "application/xml"})
-  public void submitReport(
-      @NotNull(message = "Missing handling report.") @Valid HandlingReport handlingReport) {
-    try {
-      Date completionTime =
-          new SimpleDateFormat(ISO_8601_FORMAT).parse(handlingReport.getCompletionTime());
-      VoyageNumber voyageNumber = null;
+  @Consumes(MediaType.APPLICATION_JSON)
```

Comment:
Could you kindly put back the meta-data changes? Looks like they were accidentally removed?